public/js/scenes/practiceTask.js

```diff
@@ -276,29 +267,45 @@ var effortOutcome = function() {
         trialSuccess = 1;
         // add overlap colliders so coins  on either route disappear when overlap with player body
         this.physics.add.overlap(this.player.sprite, this.gems.sprite, collectGems, null, this); 
-        // display success message for a couple of seconds,
-        feedback = this.add.text(decisionPointX, gameHeight/2-100,  
-                                 "Nice work!", {
-                                    font: "22px monospace",
-                                    fill: "#ffffff",
+
+        // create a graphics object for the background
+        const feedbackBg = this.add.graphics();
+        feedbackBg.fillStyle(0xF6F8F9, 1);
+        feedbackBg.lineStyle(2, 0xD0D5DD, 1);
+        
+        // calculate the text width and height with padding
+        const padding = { x: 20, y: 10 };
+        const width = gameWidth * 0.8;  // 80% of screen width
+        const height = 85;        
+        const yPosition = 160;
+        
+        // draw the rounded rectangle background container centered on screen
+        feedbackBg.fillRoundedRect((gameWidth - width)/2 + 120, yPosition - height, width, height, 10);
+        feedbackBg.strokeRoundedRect((gameWidth - width)/2 + 120, yPosition - height, width, height, 10);
+        
+        // add text on top of the background container
```

Comment:
Nice work with getting the `Graphics` to work. Would recommend moving it into its own `Message` class similar to what I've done with the `ProgressBar`.